### PR TITLE
Added the Hippocrates bust to medbay heirlooms, cauteries have heat, you can smoke with a space helmet

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Cauteries have Heat, half as much as igniters, which is almost the lowest value in the game.

If you wear a spaceworthy helmet you can smoke at will, even in space!

Added the Hippocrates bust to medbay heirlooms. Paramedics don't get one.

You can now swear the Hippocratic oath with these busts! It'll give you pacifism but nothing else. The process is reversible.

There's a very small chance that the Hippocrates bust was once wielded by a certain German doctor. This chance is increased for coroner heirlooms.

## Why It's Good For The Game

> Cauteries have Heat, half as much as igniters, which is almost the lowest value in the game.

I was trying to look into why you couldn't ignite bonfires with a cautery, and while I didn't find out why before I called it quits, I did notice that it didn't have a 'heat' value unlike every other ignition tool in the game, so I added one. It literally burns wounds shut.

> If you wear a spaceworthy helmet you can smoke at will, even in space!

Makes no sense that you couldn't smoke in space because there was no air.. when it was right there inside your helmet!

> Added the Hippocrates bust to medbay heirlooms. Paramedics don't get one.

I got this idea while doing the above bugfixes and I just couldn't get it out of my head, it's too funny. Paramedics don't get one because they're powergamers and laugh at the Oath, and also it doesn't feel like a paramedic thing.

> You can now swear the Hippocratic oath with these busts! It'll give you pacifism but nothing else. The process is reversible.

This is just a little fun thing you can choose to do, i think it'd be cute to see doctors swearing the oath in medbay. Plus it's reversible which can be even funnier depending on the occassion.

> There's a very small chance that the Hippocrates bust was once wielded by a certain German doctor. This chance is increased for coroner heirlooms.

This is where the PR reached true Scope Creep but I couldn't let this idea go either. It's too funny, and we DO already have precedent for references with the entrenching tool after all. The buff isn't all that special in reality, getting a medical hud while in your hand is... basically irrelevant for the roles that literally spawn with a med hud? It's just for accuracy and rule of cool's sake.

## Changelog

:cl:
fix: Cauteries have Heat, half as much as igniters, which is almost the lowest value in the game.
fix: If you wear a spaceworthy helmet you can smoke at will, even in space!
add: Added the Hippocrates bust to medbay heirlooms. Paramedics don't get one.
add: You can now swear the Hippocratic oath with these busts! It'll give you pacifism but nothing else. The process is reversible.
add: There's a very small chance that the Hippocrates bust was once wielded by a certain German doctor. This chance is increased for coroner heirlooms.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
minus FIVE? Bruh ??

**Comment by VladinXXV** [comment]
makes less sense to give it to coroner than paramedics because the coroner is literally not a doctor and does not take this oath

**Comment by carlarctg** [comment]
> makes less sense to give it to coroner than paramedics because the coroner is literally not a doctor and does not take this oath

he's the closest to TF2 medic personality-wise

**Comment by VladinXXV** [comment]
> he's the closest to TF2 medic personality-wise

you're implying this isn't the mentality of like half the cmo players

**Comment by Sealed101** [comment]
ze healing is not as rewarding as ze hurting

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -617,6 +617,92 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+	if(!iscarbon(user))
+		to_chat(user, span_warning("You remember how the Hippocratic Oath specifies 'my fellow human beings' and realize that it's completely meaningiess to you."))
+		return
+
+	if(reference)
+		to_chat(user, span_warning("As you prepare yourself to swear the Oath, you realize that doing so on a blood-caked bust is probably not a good idea."))
+		return
+
+	if(HAS_TRAIT_FROM(user, TRAIT_PACIFISM, type))
+		to_chat(user, span_warning("You've already sworn a bow. You start preparing to rescind it..."))
```
Comment (carlarctg) [inline code/game/objects/items/weaponry.dm:670]:
```suggestion
		to_chat(user, span_warning("You've already sworn a vow. You start preparing to rescind it..."))
```

you stupid brute

**Comment by CRITAWAKETS** [comment]
smoking with internals should fill your tank with CO2

**Comment by SimplySmiley123** [comment]
> > he's the closest to TF2 medic personality-wise
> 
> you're implying this isn't the mentality of like half the cmo players

Or half the crew in general.

**Comment by ZeroKelvn** [comment]
> There's a very small chance that the Hippocrates bust was once wielded by a certain German doctor. This chance is increased for coroner heirlooms.

If the bust doesn't give you the medical HUD effect, reference would probably work better if it do, though not sure if that would add a balance tag

**Comment by VladinXXV** [comment]
> If the bust doesn't give you the medical HUD effect, reference would probably work better if it do, though not sure if that would add a balance tag

Reading the "Why it's good for the game" section will answer your curiosity.

**Comment by ZeroKelvn** [comment]
> > If the bust doesn't give you the medical HUD effect, reference would probably work better if it do, though not sure if that would add a balance tag
> 
> Reading the "Why it's good for the game" section will answer your curiosity.

Wow I really did just blindsided that huh

**Comment by optimumtact** [comment]
atomise unrelated changes.

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -617,6 +617,92 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+	..()
+	if(HAS_TRAIT_NOT_FROM(user, TRAIT_MEDICAL_HUD, type))
+		return
+	var/datum/atom_hud/our_hud = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	our_hud.hide_from(user)
+	REMOVE_TRAIT(user, TRAIT_MEDICAL_HUD, type)
+
+/obj/item/statuebust/hippocratic/attack_self(mob/user)
+	if(!iscarbon(user))
+		to_chat(user, span_warning("You remember how the Hippocratic Oath specifies 'my fellow human beings' and realize that it's completely meaningiess to you."))
```
Comment (Arturlang) [inline code/game/objects/items/weaponry.dm:662]:
```suggestion
		to_chat(user, span_warning("You remember how the Hippocratic Oath specifies 'my fellow human beings' and realize that it's completely meaningless to you."))
```

**Comment by dieamond13** [comment]
you can already smoke in space if you smoke space cigs

**Comment by Jackal-boop** [comment]
Wouldn't smoking a cigarette in your pure oxygen helmet cause a boom

**Comment by Jacquerel** [comment]
> atomise unrelated changes.

**Pull request closed without merging**